lib/between_meals/repo/hg.rb

```diff
@@ -167,37 +167,37 @@ def parse_status(changes)
 
         changes.lines.map do |line|
           case line
-          when /^A (\S+)$/
+          when /^A (.+)$/
```

Comment:
does `hg` not use tabs?

Reply:
Nah.  It's whatever hex 20 is, which is probably a space.

Reply:
it seems especially odd to me to "support" this only in one backend. :/

Reply:
I don't follow.  Support what?  As is, this handles spaces in all three backends.

Reply:
On the bright side, I guess I can't find any scenario on which there's more than 2 fields.

That said we probably shuold still split

Reply:
well, not reliably if there's a 2nd or 3rd field...

Reply:
Are you saying you want to future-proof for when a 2nd or 3rd field might exist for hg / svn?  Given that it'll most likely have a different character as the 1st field, that feels like something that should be fielded when the time comes.

Reply:
Same reason you wanted to use tabs for git instead of just assuming rest-of-line. They could add a .. star at the end of svnprops were set, for example. Who knows. As opposed to in git where we now can because there's a field separator.

Nonetheless we can't solve that problem today.

What we should do, however, is still use split instead of regex... most to be consistent across the plugins, but also because it's a lot faster. `line.split(nil, 2)`